src/frontend/komponenter/Fagsak/Vedtak/VedtakBegrunnelserTabell/VedtaksperioderMedBegrunnelser/VedtaksperioderMedBegrunnelser.tsx

```diff
@@ -33,13 +37,39 @@ const VedtaksperioderMedBegrunnelser: React.FC<IVedtakBegrunnelserTabell> = ({
         return <AlertStripeFeil>Klarte ikke å hente inn begrunnelser for vedtak.</AlertStripeFeil>;
     }
 
+    const avslagOgResterende = partition(
+        vedtaksperiode => vedtaksperiode.type === Vedtaksperiodetype.AVSLAG,
+        vedtaksperioderMedBegrunnelser
+    );
+
     return vedtaksperioderMedBegrunnelser.length > 0 ? (
         <>
             <OverskriftMedHjelpetekst
                 overskrift={'Begrunnelser i vedtaksbrev'}
                 hjelpetekst={'Her skal du sette begrunnelsestekster for fortsatt innvilgelse'}
             />
-            {vedtaksperioderMedBegrunnelser.map(
+            {avslagOgResterende[1].map(
+                (vedtaksperiodeMedBegrunnelser: IVedtaksperiodeMedBegrunnelser) => (
+                    <VedtaksperiodeMedBegrunnelserProvider
+                        key={vedtaksperiodeMedBegrunnelser.id}
+                        fagsak={fagsak}
+                        åpenBehandling={åpenBehandling}
+                        vedtaksperiodeMedBegrunnelser={vedtaksperiodeMedBegrunnelser}
+                    >
+                        <VedtaksperiodeMedBegrunnelserPanel
+                            vedtaksperiodeMedBegrunnelser={vedtaksperiodeMedBegrunnelser}
+                        />
+                    </VedtaksperiodeMedBegrunnelserProvider>
+                )
+            )}
+
+            <OverskriftMedHjelpetekst
+                overskrift={'Begrunnelser for avslag i vedtaksbrev'}
+                hjelpetekst={
+                    'Her har vi hentet begrunnelsestekster for avslag som du har satt i vilkårsvurderingen.'
+                }
+            />
+            {avslagOgResterende[0].map(
```

Comment:
Om vi lager et objekt så kunne vi også bygget inn overskrift og hjelpetekst og bare kjørt map på verdiene på dette objektet. Eventuelt så tenker jeg hvertfall vi bør trekke ut `<VedtaksperiodeMedBegrunnelserProvider>...</>` som er felles for begge mappene som skjer her.